### DISCUSSION TITLE
NanoPC T4 images not starting

### DISCUSSION THREAD

**Discussion by SvenGeorgi** [johang/sd-card-images#137] [Q&A]
Tried different bookworm images on my device NanoPC T4 to boot from SD card but none is booting at all. I tried images from armbian and the last one from Johang following this procedue:
wget https://dl.sd-card-images.johang.se/boots/2024-06-01/boot-nanopc_t4.bin.gz
wget https://dl.sd-card-images.johang.se/debians/2024-06-17/debian-bookworm-arm64-uch4ai.bin.gz
zcat boot-nanopc_t4.bin.gz debian-bookworm-arm64-uch4ai.bin.gz > sd-card.img
dd if=sd-card.img of=/dev/sda
Writing the image to SD card completes without any error and the data are consistent. But my NanaoPC T4 does not show any sign to start any boot sequence. The HW is OK because my older systems are booting from SDcard witout any issue. Any ideas what I am doing wrong?

**Comment by johang** [comment]
Connect to the UART console and show the complete log output during boot.
